package.json

```diff
@@ -1,34 +1,53 @@
 {
   "name": "oauth2-server",
   "description": "Complete, framework-agnostic, compliant and well tested module for implementing an OAuth2 Server in node.js",
-  "version": "3.0.0",
+  "version": "3.0.1",
   "keywords": [
     "oauth",
     "oauth2"
   ],
   "contributors": [
-    { "name": "Thom Seddon", "email": "thom@seddonmedia.co.uk" },
-    { "name": "Lars F. Karlström" , "email": "lars@lfk.io" },
-    { "name": "Rui Marinho", "email": "ruipmarinho@gmail.com" },
-    { "name" : "Tiago Ribeiro", "email": "tiago.ribeiro@gmail.com" },
-    { "name": "Michael Salinger", "email": "mjsalinger@gmail.com" },
-    { "name": "Nuno Sousa" },
-    { "name": "Max Truxa" }
+    {
+      "name": "Thom Seddon",
+      "email": "thom@seddonmedia.co.uk"
+    },
+    {
+      "name": "Lars F. Karlström",
+      "email": "lars@lfk.io"
+    },
+    {
+      "name": "Rui Marinho",
+      "email": "ruipmarinho@gmail.com"
+    },
+    {
+      "name": "Tiago Ribeiro",
+      "email": "tiago.ribeiro@gmail.com"
+    },
+    {
+      "name": "Michael Salinger",
+      "email": "mjsalinger@gmail.com"
+    },
+    {
+      "name": "Nuno Sousa"
+    },
+    {
+      "name": "Max Truxa"
+    }
   ],
   "main": "index.js",
   "dependencies": {
-    "basic-auth": "1.1.0",
-    "bluebird": "3.5.0",
-    "lodash": "4.17.4",
-    "promisify-any": "2.0.1",
-    "statuses": "1.3.1",
-    "type-is": "1.6.15"
+    "basic-auth": "^2.0.0",
+    "bluebird": "^3.5.0",
+    "lodash": "^4.17.4",
+    "promisify-any": "^2.0.1",
+    "statuses": "^1.3.1",
+    "type-is": "^1.6.15"
   },
   "devDependencies": {
-    "jshint": "2.9.4",
```

Comment:
This will still cause security warnings from npm audit. jshint has old dependencies and hasn't released a version in a while - it's probably not maintained anymore. I recommend replacing with eslint.